examples/Python/MR/Gadgetron/run_all.py

```diff
@@ -30,6 +30,7 @@
     args = ''
     for a in range(1, narg):
         args += ' ' + sys.argv[a]
-    err = os.system(sys.executable + ' ' + i + args)
+    exe = 'python ' + i + args
```

Comment:
This will fail on systems where the executable is not called `python` (such as our new VM, where it is `python3`). Instead, use quotes, see https://stackoverflow.com/questions/6977215/os-system-to-invoke-an-exe-which-lies-in-a-dir-whose-name-contains-whitespace

Reply:
@KrisThielemans: Had to do it this way because neither the solution you quoted nor any other solutions I found on the Internet worked. Will think about yet another solution.

Reply:
what did `sys.executable` report?

Reply:
@KrisThielemans 'C:\Program' is not recognized as an internal or external command, operable program or batch file.

Reply:
@KrisThielemans this works:
```
    exe = os.path.split(sys.executable)[1] + ' ' + i + args
    err = os.system(exe)
```

Reply:
this is effectively like calling `os.path.basename`. I think it will fail when the directory is not on the path.

it's possible there are some issues with spaces and quotes. I think it's important to understand what `sys.executable` is exactly.

what did you try with the quotes?

Reply:
@KrisThielemans "directory is not on the path" - do you mean Python executable directory? Well, then using Python would be very inconvenient, would it not?

Of what I was trying, I remember
```
    err = os.system('"' + exe + '"')
```
and (somebody suggested this weird solution specifically for Windows)
```
    err = os.system('""' + exe + '""')
```
and
```
    err = subprocess.call([exe])
```
all with
```
    exe = sys.executable + ' ' + i + args
```

Reply:
>  "directory is not on the path" - do you mean Python executable directory? Well, then using Python would be very inconvenient, would it not?

you don't know how they call it. From jupyter or whatever, it might not be in the path. Or there might be another python in the path which conflicts.

Please try
```python
exe = '"' + sys.executable + '"  ' + i + args
```
the quotes need to be around the executable only

Reply:
@KrisThielemans it worked, thanks!